1-js/02-first-steps/04-variables/article.md

```diff
@@ -1,330 +1,331 @@
 # Variables
 
-Most of the time, a JavaScript application needs to work with information. Here are two examples:
-1. An online shop -- the information might include goods being sold and a shopping cart.
-2. A chat application -- the information might include users, messages, and much more.
+În cea mai mare parte a timpului, o aplicație JavaScript are nevoie de informație pentru a funcționa. Două exemple sunt:
+1. Un magazin online -- informația poate include bunurile vândute și produsele din coș.
+2. O aplicație de mesaje -- informația poate include utilizatorii, mesajele și multe altele.
 
-Variables are used to store this information.
+Variabilele sunt folosite pentru a stoca această informație.
 
-## A variable
+## O variabilă
 
-A [variable](https://en.wikipedia.org/wiki/Variable_(computer_science)) is a "named storage" for data. We can use variables to store goodies, visitors, and other data.
+O [variabilă](https://ro.wikipedia.org/wiki/Variabil%C4%83) este un „spațiu de stocare denumit” pentru date. Putem folosi variabile pentru a stoca bomboane, vizitatori și alte date.
 
-To create a variable in JavaScript, use the `let` keyword.
+Pentru a crea o variabilă în JavaScript, folosim termenul `let`.
 
-The statement below creates (in other words: *declares* or *defines*) a variable with the name "message":
+Codul de mai jos creează (în alte cuvinte, *declară* sau *definește*) o variabilă cu numele „mesaj”:
 
 ```js
-let message;
+let mesaj;
 ```
 
-Now, we can put some data into it by using the assignment operator `=`:
+Acum, putem pune date în ea utilizând operatorul de atribuire `=`:
 
 ```js
-let message;
+let mesaj;
 
 *!*
-message = 'Hello'; // store the string
+mesaj = 'Bună'; // stocăm șirul de caractere
 */!*
 ```
 
-The string is now saved into the memory area associated with the variable. We can access it using the variable name:
+Șirul de caractere este acum salvat în zona de memorie asociată cu variabila. O putem accesa utilizând numele variabilei:
 
 ```js run
-let message;
-message = 'Hello!';
+let mesaj;
+mesaj = 'Bună!';
```

Comment:
In acest caz, continutul din variabila poate fi pastrat. Variabila `message` nu ar trebui sa fie schimbata cu `mesaj`.